src/app/cake/cake.class.ts

```diff
@@ -0,0 +1,26 @@
+
+export enum CakeStatus {
+  Available = "AVAILABLE",
+  LastUnits = "LASTUNITS",
+  OutOfStock = "OUTOFSTOCK"
+} 
+
+export interface Cake { }
+
+export class CarrotCake implements Cake{ 
```

Comment:
`CarrotCake` ? 🤔

If this is the main class, you should name it `Cake`

Reply:
Done ✅